src/vm-repair/azext_vm_repair/_help.py

```diff
@@ -77,3 +77,18 @@
           text: >
             az vm repair list-scripts --preview "https://github.com/haagha/repair-script-library/blob/master/map.json"
 """
+
+helps['vm repair reset-nic'] = """
+    type: command
+    short-summary: Resets the network interface stack on the VM guest OS. https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/reset-network-interface
```

Comment:
Please use the first person voice in the help message
```suggestion
    short-summary: Reset the network interface stack on the VM guest OS. https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/reset-network-interface
```